README.md

```diff
@@ -201,7 +224,7 @@ $ curl -Lo containerbuddy-0.0.4.tar.gz \
 $ tar -xf containerbuddy-0.0.4.tar.gz
 $ cp ./containerbuddy ./nginx/opt/containerbuddy/
 $ cp ./containerbuddy ./app/opt/containerbuddy/
-./start.sh -p example -f docker-compose-local.yml
+./run.sh -p example -f docker-compose-local.yml
```

Comment:
> I can fix up this in the readme, but I don't have a good way of creating the non-local compose for etcd since it has some external dependencies to which I don't have access.

It really should just be the same as the one in the `consul` directory with the appropriate `etcd` vs `consul` sections swapped.

> Aside - the readme make reference to a specific version of Containerbuddy for the tarball. Should this be updated when the version gets bumped?

Ugh. When we decide to cut a 1.0 release that problem will go away because we can link to "latest". But for now, yeah.